From e4482f9212990ffbfcf22e72eafb2bd98be94b82 Mon Sep 17 00:00:00 2001
From: Jon Medhurst <tixy@linaro.org>
Date: Wed, 11 Jul 2012 16:14:29 +0100
Subject: [PATCH 23/28] ARM: vexpress: Add CPU clock-frequencies to TC2
 device-tree

Commit 4f1948004608c4f6c04e704e14c40abf0a209f35 from
git://git.linaro.org/kernel/linux-linaro-tracking.git

Signed-off-by: Jon Medhurst <tixy@linaro.org>
Signed-off-by: Guoqing Jiang <Guoqing.Jiang@windriver.com>
---
 arch/arm/boot/dts/vexpress-v2p-ca15-tc2.dts |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/vexpress-v2p-ca15-tc2.dts b/arch/arm/boot/dts/vexpress-v2p-ca15-tc2.dts
index 2d94277..96ac1bc 100644
--- a/arch/arm/boot/dts/vexpress-v2p-ca15-tc2.dts
+++ b/arch/arm/boot/dts/vexpress-v2p-ca15-tc2.dts
@@ -88,6 +88,7 @@
 			reg = <0>;
 			cluster = <&cluster0>;
 			core = <&core0>;
+			clock-frequency = <1000000000>;
 		};
 
 		cpu1: cpu@1 {
@@ -96,6 +97,7 @@
 			reg = <1>;
 			cluster = <&cluster0>;
 			core = <&core1>;
+			clock-frequency = <1000000000>;
 		};
 
 		cpu2: cpu@2 {
@@ -104,6 +106,7 @@
 			reg = <0x100>;
 			cluster = <&cluster1>;
 			core = <&core2>;
+			clock-frequency = <800000000>;
 		};
 
 		cpu3: cpu@3 {
@@ -112,6 +115,7 @@
 			reg = <0x101>;
 			cluster = <&cluster1>;
 			core = <&core3>;
+			clock-frequency = <800000000>;
 		};
 
 		cpu4: cpu@4 {
@@ -120,6 +124,7 @@
 			reg = <0x102>;
 			cluster = <&cluster1>;
 			core = <&core4>;
+			clock-frequency = <800000000>;
 		};
 	};
 
-- 
1.7.9.7

